Rakefile

```
@@ -5,4 +5,4 @@ require File.expand_path("config/application", __dir__)
 
```

I think most projects lint as part of their default rake task - you could add:

```suggestion
require 'rubocop/rake_task'

Rails.application.load_tasks

RuboCop::RakeTask.new
task default %i[spec rubocop]
```